### ISSUE TITLE
[Feature] Connect with iOS & Android template naming convention

### ISSUE THREAD

**Issue by MarkPieszak** [angular/vscode-ng-language-service#148]
Although Nativescript templates are referenced only by the regular file name:

```typescript
@Component({
    selector:    'my-creation',
    templateUrl: 'my-creation.component.html', // <--
})
```

In Nativescript, Components can split their view templates by `iOS` & `Android`, by adding that name before `.html`.  This causes language services to lose it's connection to the tempate & component class.

![image](https://user-images.githubusercontent.com/2574412/29437933-adb4f9b0-8381-11e7-8279-d68f8e966f2a.png)

Would it be possible to have it look for `.ios.html` & `.android.html` when creating the connection between the two files?

cc: @chuckjaz @NathanWalker @jlooper

**Comment by chuckjaz** [comment]
I dislike the idea of burning in such a convention. Maybe some sort of configuration such as

```
  templateExtensions: {
    "*.html" : ["*.ios.html", "*.android.html"]
  }
```

which would mean to treat `foo.ios.html` as if it were `foo.html`.

**Comment by MarkPieszak** [comment]
I agree, especially since it's only really needed in this scenario, but it'd be amazing if we could configure it! Thank you Chuck! 👏 

Hopefully we can think of some way we could get {N} to automatically setup that configuration for new projects as well.

**Comment by drewpayment** [comment]
Just wondering if this is still on the radar. This will be an extremely helpful feature to have for NativeScript development as an Angular developer who is learning NS.

**Comment by kyliau** [comment]
We'll look into a way to provide custom template path resolution, but it'll take a while. Probably after Angular v10.

**Comment by miladfm** [comment]
any update on this feature?

**Comment by kyliau** [comment]
The new Ivy-native language service released in [v11.1.0](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0) no longer has built-in assumption for `templateUrl`. The suffix of the filename does not affect template resolution in any way. Please try it out and if you run into similar problem, please open a new issue. I'll close this for now.
